Web/ForgeTrust.Runnable.Web.RazorDocs/wwwroot/css/app.css

```
@@ -0,0 +1,321 @@
+    margin-top: 0.15rem;
+}
+
+.docs-content .doc-overload[open] > summary::before {
+    content: "-";
+}
+
+.docs-content .doc-overload > summary code {
+    color: #e2e8f0;
+    font-weight: 500;
+    white-space: normal;
+    word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)